bip-0009.mediawiki

```diff
@@ -0,0 +1,167 @@
+<pre>
+  BIP: 9
+  Title: Version bits with timeout and delay
+  Author: Pieter Wuille <pieter.wuille@gmail.com>, Peter Todd <pete@petertodd.org>, Greg Maxwell <greg@xiph.org>, Rusty Russell <rusty@rustcorp.com.au>
+  Status: Draft
+  Type: Informational Track
+  Created: 2015-10-04
+</pre>
+
+==Abstract==
+
+This document specifies a proposed change to the semantics of the 'version' field in Bitcoin blocks, allowing multiple backward-compatible changes (further called called "soft forks") being deployed in parallel. It relies on interpreting the version field as a bit vector, where each bit can be used to track an independent change. These are tallied each retarget period. Once the consensus change succeeds or times out, there is a "fallow" pause after which the bit can be reused for later changes.
+
+==Motivation==
+
+BIP 34 introduced a mechanism for doing soft-forking changes without predefined flag timestamp (or flag block height), instead relying on measuring miner support indicated by a higher version number in block headers. As it relies on comparing version numbers as integers however, it only supports one single change being rolled out at once, requiring coordination between proposals, and does not allow for permanent rejection: as long as one soft fork is not fully rolled out, no future one can be scheduled.
```

Comment:
฿tcDrak <notifications@github.com> writes:
> s/without predefined flag/without a predefined flag/

Applied, with the others.  Thanks.

Cheers,
Rusty.